### ISSUE TITLE
Incorrect error message for "stroustrup" brace-style

### ISSUE THREAD

**Issue by phraemer** [eslint/eslint#4143]
Same as old closed issue https://github.com/eslint/eslint/issues/719

v1.6.0

    4:12  error  Closing curly brace appears on the same line as the subsequent block  brace-style
    ? 1 problem (1 error, 0 warnings)

```javascript
  "rules": {
    "brace-style": [2, "stroustrup", { "allowSingleLine": true }],
    "block-scoped-var": 2
  }

{
    if (true) {
        //
    } else { // <- error here after the brace
        //
    }
}

```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
This message seems correct to me, what did you expect it to be?

**Comment by phraemer** [comment]
Sorry, I shouldn't have copy-pasted the title from the other issue.
What I meant was that it doesn't seem to really an error.
As far I can tell that is the correct stroustroup style and it is reporting it as an error.
Or am I missing something?

**Comment by nzakas** [comment]
Stroustrop style would be this:

```js
    if (true) {
        //
    } 
    else { // <- error here after the brace
        //
    }
```

**Comment by ilyavolodin** [comment]
@phraemer Your example is written using 1tbs. Stroustrap requires closing brace on a separate line. So it looks like the error is correct.

**Comment by phraemer** [comment]
Ok, thanks for the clarification. My mistake!
